registrations/utils.py

```diff
@@ -194,6 +194,14 @@ def get_web_store_api_error_message(response):
     return error
 
 
+def get_web_store_api_refunds_error_messages(response_json):
+    if not response_json.get("errors"):
+        error_message = _("Unknown Talpa web store refund error")
+        return [str(error_message)]
```

Comment:
Is it actually even possible that this gets executed. It seems this function will not be called with `response_json.get("errors")` being falsy 🤔

Reply:
True, it's never executed in the current cases where it's used. I modified the function so that it doesn't have these unused lines.